chance.js

```diff
@@ -926,6 +927,22 @@
                 min: min,
                 max: max
             });
+        } if (options && (options.minAge || options.maxAge)) {
+            testRange(options.minAge > options.maxAge, "Chance: minAge cannot be greater than maxAge.");
+
+            var minAge = typeof options.minAge !== "undefined" ? options.minAge : 1;
+            var maxAge = typeof options.maxAge !== "undefined" ? options.maxAge : 100;
+
+            var minDate = new Date(now.getFullYear() - maxAge - 1, now.getMonth(), now.getDate());
```

Comment:
You create the variable `currentYear` above but then you recompute it here. Please re-use the variable you created on `918` rather than recalling `now.getFullYear()`